### PR TITLE
Added HitNormal to Highspeed Ranger

### PR DESCRIPTION
**Pull request by suunrider**

### CODE REVIEW

**Comment by Divran** [comment]
Looks fine to me. @AbigailBuccaneer or @Nebual ?

**Comment by Nebual** [comment]
I don't really know enough about the highspeed stuff; I can confirm this passes basic lua sanity checks!

**Comment by AbigailBuccaneer** [comment]
LGTM :ship:

File: lua/entities/gmod_wire_hsranger/init.lua [hunk 1]
```diff
@@ -18,7 +18,9 @@ local MODEL = Model( "models/jaanus/wiretool/wiretool_range.mdl" )
 	5 = SkewY
 	6 = Trace Water
 	
-	7 = unused
```
Comment (TomyLobo) [inline lua/entities/gmod_wire_hsranger/init.lua]: Hmm, did it say unused for a reason?
Was there something else in slot 7 in the past, maybe?
Or is this a zero terminator?

File: lua/entities/gmod_wire_hsranger/init.lua [hunk 2]
```diff
@@ -111,7 +117,7 @@ function ENT:WriteCell( Address, Value )
 	if ( Address == 0 and Value > 0 ) then
 		self:Trace()
 		return true
-	elseif ( Address >= 2 and Address <= 7 ) then
+	elseif ( Address >= 2 and Address <= 6 ) then
```
Comment (TomyLobo) [inline lua/entities/gmod_wire_hsranger/init.lua]: 6? shouldn't this be 9?

File: lua/entities/gmod_wire_hsranger/init.lua (hunk 1 above)
Comment (Nebual) [inline lua/entities/gmod_wire_hsranger/init.lua]: Low level hardware protocols often seem to have unused reserved spaces for future development, like TCP headers. Despite being totally unnecessary in Lua, I think they're here for flavour, and to nicely round out the half byte of 0-7.

File: lua/entities/gmod_wire_hsranger/init.lua (hunk 2 above)
Comment (TomyLobo) [inline lua/entities/gmod_wire_hsranger/init.lua]: @suunrider @AbigailBuccaneer are you gonna fix that ReadCell?

**Pull request merged**